_includes/k8s/bestpractices/swarm-nodes.md

```diff
@@ -0,0 +1,5 @@
+## StorageOS in DockerEE
+
+It is recommended to use Kubernetes nodes to provision Stateful workloads as
```

Comment:
I think we should move this comment to the root of the Docker EE section, or duplicate it there. Let's not bury it in a Best Practice but shout it clearly.

Reply:
Agreed